feat: Added csdiff and git packages

jperezdealgaba opened the pull request:
csdiff and git packages are installed in order to be used by SAST tasks to parse the results and generate fingerprinting.

This will be used by future SAST tasks provided by the OpenScanHub team, for example: https://issues.redhat.com/browse/OSH-737

jperezdealgaba commented:
@ralphbean Would you mind giving a review on this?

jsztuka reviewed (approved):
lgtm

jperezdealgaba commented:
@ralphbean @14rcole @Josh-Everett Could any of you review it and approve it/comment it? Thank you!

ralphbean commented:
/ok-to-test

jperezdealgaba commented:
@ralphbean Would you mind enabling the last CI test? I am not able to trigger it/merge this

dirgim commented:
@jperezdealgaba if you rebase your commit on the latest `main`, the Red Hat Konflux job issue should get resolved.

jperezdealgaba commented:
@dirgim I rebased the PR and GitHub is still showing that I need the approval from one maintainer for the workflow:

![Screenshot 2024-09-09 at 15 58 16](https://github.com/user-attachments/assets/16fd6a3f-7bb5-44ee-a690-cf8fe0791080)

P.S.: I added the installation of the git packages to this PR as it will be also needed. I hope it is not a problem

jperezdealgaba commented:
All checks passed! I have no merge rights in this repo

dirgim reviewed (approved):
+1
